spec/rspec_matchers/schema_spec.rb

```diff
@@ -0,0 +1,127 @@
+require 'rails_helper'
+
+module ActiveModelSerializers
+  module Test
+    module SchemaTest
+      class MyController < ActionController::Base
+        def index
+          render json: profile
+        end
+
+        def show
+          index
+        end
+
+        def name_as_a_integer
+          profile.name = 1
+          index
+        end
+
+        def render_using_json_api
+          render json: profile, adapter: :json_api
+        end
+
+        def invalid_json_body
+          render json: ''
+        end
+
+        private
+
+        def profile
+          @profile ||= Profile.new(name: 'Name 1', description: 'Description 1', comments: 'Comments 1')
+        end
+      end
+    end
+  end
+end
+
+describe ActiveModelSerializers::Test::SchemaTest::MyController, type: :controller do
+  routes { Routes }
+
+  include ActiveModelSerializers::RSpecMatchers::Schema
```

Comment:
So you want it that by including `require 'active_model_serializers/rspec` `ActiveModelSerializers::RSpecMatchers::Schema` is included on all controller and request specs?